test/functionalTest/cases/0947_v2_list_all_entities/GET_v2_entities.test

```diff
@@ -277,13 +277,16 @@ Date: REGEX(.*)
 03. GET /v2/entities
 ====================
 HTTP/1.1 200 OK
-Content-Length: 232
+Content-Length: 288
 Content-Type: application/json
 Date: REGEX(.*)
 
 [
     {
-        "A0": 255, 
+        "A0": {
+            "type": "number", 
+            "value": "255"
```

Comment:
Moreover, using POST /v2/entities.for creating entities would allow to use actual number, thus having:

```
"A0": {
   "type": "number",
   "value": 255
}
```